README.md

```diff
@@ -104,6 +104,44 @@ Alternatively, use online services (like Google Colab):
 
 - [List of Online Services](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Online-Services)
 
+### Running with Docker
+
+#### a) Run with Docker Compose
+DISCLAIMER: This currently only works with NVIDIA GPUs
+
+You need to have [Docker](https://www.docker.com/) installed on your system. Then clone this repository and execute `docker compose -f docker/compose.yml up` in the root path of the repository. The first time you execute this command will take a long time as all the dependencies are installed. Subsequent runs of the command should start up the webui pretty much instantly. To stop the webui press CTRL+C and wait a few seconds.
+
+Models are provided to the Docker container using a bind mount. This means that if you add a new model to the models directory it should be available in the webui after a checkpoint refresh without needing to rebuild or restart the container.
+
+The server will be accessible at [localhost:7860](localhost:7860)
+
+#### b) Run with docker CLI
+```
+git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui
+cd stable-diffusion-webui
+export TAG=v1.10.0
+docker build -t stable-diffusion-webui:${TAG}  -f docker/Dockerfile .
+docker run --gpus all -d -p 7860:7860  stable-diffusion-webui:${TAG}

```

Comment:
(A) Minor, but formatting is a little weird here, double-spacing left in.
(B) Minor, but also the tag usage seems a bit unusual.
(C) the `-v` for models isn't really optional, it should be included not a "if you happen to already have models". This line should have all the same contents as the compose does by default.

Reply:
thank you.

hmm, about (c) , after careful thought, you are right .even without pre-download model files in volume or in container, the webui.sh will still download image when containers up